src/main/src/main/java/org/geoserver/catalog/impl/FeatureTypeInfoImpl.java

```diff
@@ -239,4 +242,15 @@ public void setCqlFilter(String cqlFilter) {
         this.cqlFilter = cqlFilter;
         this.filter = null;
     }
+
+    @Override
+    public boolean getEncodeCoordinatesMeasures() {
+        // by default encoding of coordinates measures is activated
+        return encodeCoordinatesMeasures == null ? true : encodeCoordinatesMeasures;
```

Comment:
Not sure if it should default to true. In GML that will break standard clients. In GeoJSON it might confuse some.

Reply:
Making coordinates measures not encoded by default, done.